pkg/flare/config_check.go

```diff
@@ -43,13 +44,11 @@ func PrintConfigCheck(w io.Writer, cr integration.ConfigCheckResponse, withDebug
 			}
 		}
 		if len(cr.Unresolved) > 0 {
-			fmt.Fprintf(w, "\n=== %s Configs ===\n", color.YellowString("Unresolved"))
-			for ids, configs := range cr.Unresolved {
-				fmt.Fprintf(w, "\n%s: %s\n", color.BlueString("Auto-discovery IDs"), color.YellowString(ids))
-				fmt.Fprintf(w, "%s:\n", color.BlueString("Templates"))
-				for _, config := range configs {
-					fmt.Fprintln(w, config.String())
-				}
+			fmt.Fprintf(w, "\n=== %s configs (matched and unmatched) ===\n", color.MagentaString("Collected"))
```

Comment:
I changed the color because yellow can give the impression that it’s a warning, but this is just informational.